src/python/pants/backend/python/python_artifact.py

```diff
@@ -18,8 +18,9 @@ class UnsupportedArgument(Exception):
 
     def __init__(self, **kwargs):
         """
-    :param kwargs: Passed to `setuptools.setup
-       <https://pythonhosted.org/setuptools/setuptools.html>`_."""
+        :param kwargs: Passed to `setuptools.setup
+          <https://pythonhosted.org/setuptools/setuptools.html>`_.
+       """
```

Comment:
This closing `"""` looks like it needs an extra space to indent.

Reply:
Good catch!

Reply:
Fine with fixing as a follow-up though.